website/common/src/enums/kafka-topics.enum.ts

```diff
@@ -7,6 +7,7 @@ enum KAFKA_TOPICS {
   PROGRAM_ALL_USER = 'program.all.user',
   MESSAGE_ALL = 'message.all',
   MESSAGE_DATA = 'message.data',
+  MESSAGE_UPDATE_DATA = 'message.update.data',
```

Comment:
@osipov-mit 
I have thought about the same topic name. There are some reasons why I avoid setting the `message.update.payload`.

- I chose `message.update.data` because it is more generally. Also with we get benefits to extend with other `fields` our `UpdateMessageParams` `interface` in the future.
- If we set `message.update.payload` It means we should create a new `Kafka topic` for every update `field` the `message` entity in the future.